@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -429,11 +525,11 @@ export function getMaterialConstructor(technique: Technique): MaterialConstructo
  */
 export function evaluateBaseColorProperty(
     technique: Technique,
-    zoomLevel?: number
+    zoomLevel: number
 ): number | undefined {
     const baseColorProp = getBaseColorProp(technique);
     if (baseColorProp !== undefined) {
-        return evaluateColorProperty(baseColorProp, zoomLevel);
+        return getColorPropertyValueSafe(baseColorProp, undefined, zoomLevel);
```

Comment:
Is the check `baseColorProp !== undefined` still necessary? I think `getColorPropertyValueSafe` already checks if the property is `null` or `undefined`.

Reply:
Agree, fixed.